src/stream/src/common/table/state_table.rs

```diff
@@ -1025,7 +1029,29 @@ where
     }
 
     pub async fn commit(&mut self, new_epoch: EpochPair) -> StreamExecutorResult<()> {
+        self.commit_inner(new_epoch, false).await
+    }
+
+    pub async fn commit_with_enable_consistent_op(
```

Comment:
What about changing its name to `commit_with_switch_consistent_op`? And let `commit_inner` accept a parameter `Option<bool>` instead of bool.

Reply:
Good idea! I've changed the semantic from `enable` to `switch`.